### ISSUE TITLE
Warning about density_lion_decode_bitmasks

### ISSUE THREAD

**Issue by nemequ** [centaurean/density#20]
```
kernel_lion_decode.c:90:47: warning: ‘density_lion_decode_bitmasks’ is static but used in inline function ‘density_lion_decode_read_4bits_from_signature’ which is not static
                 result |= (state->signature & density_lion_decode_bitmasks[overflowBits]) << (density_bitsizeof(density_lion_signature) - state->shift);   // Add bits from the new signature
                                               ^
```

**Comment by gpnuma** [comment]
This is a GCC specific warning, on CLANG it is not triggered.

**Comment by gpnuma** [comment]
Fixed in 561c226f91131c04bd9ec977479d6804d72035fe
